docs_user/assemblies/assembly_storage-requirements.adoc

```diff
@@ -10,7 +10,7 @@ When looking into the storage in an {rhos_prev_long} ({OpenStackShort}) deployme
 two kinds, the storage requirements of the services themselves and the
 storage used for the {OpenStackShort} users that the services will manage.
 
-These requirements may drive your {rhocp_long} node selection, as mentioned above,
+These requirements may drive your {rhocp_long} node selection
 and may require you to do some preparations on the {OpenShiftShort} nodes before
 you can deploy the services.
 
```

Comment:
Has the TODO comment been addressed? 
//*TODO: Galera, RabbitMQ, Swift, Glance, etc.*